modules/services/MapillaryService.js

```diff
@@ -1,6 +1,8 @@
 import { select as d3_select } from 'd3-selection';
 import { Tiler } from '@rapid-sdk/math';
 import { VectorTile } from '@mapbox/vector-tile';
+import { osmNode } from '../osm/node';
```

Comment:
Since we're updating this service to be able to convert from mapillary objects to OSM Nodes, it has to know about them!